src/tools/rest-frontend/resources/assets/html/index.html.in

```diff
@@ -34,7 +34,11 @@
                 }
 
             };
+
         </script>
+        <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
```

Comment:
For me this line looks like it could cause the problem I have with cdn.jsdelivr.net (destroying the whole homepage).

Reply:
Maybe docsearch should be added in the package.json, if they provide an npm package?

Reply:
I do not know if this would fix the problem. @Namoshek can you maybe take a look?

Reply:
For me the website works fine. I guess one of your anti-tracker browser extensions blocks the cdn?. I suggest adding `docsearch` to `package.json` and linking it in the `concat.vendor` target of our `Gruntfile.js` to pack it with the other vendor packages in a neat bundle:

https://github.com/ElektraInitiative/libelektra/blob/e551ccb5c0bed791dbe4fee7941b9cc757c339e2/src/tools/rest-frontend/Gruntfile.js#L139-L162

Reply:
Thank you for the quick help!

Yes, it is caused by blocking the site. But in any case it should not make the whole site fail (it is obviously ok that the search will not work).